### DISCUSSION TITLE
Rajouter des friandises aux effets aléatoires

### DISCUSSION THREAD

**Discussion by QQtin3** [Crownicles/Crownicles#672] [Idées acceptées / déjà implémentées]
On pourrait imaginer pouvoir obtenir des friandises aléatoires qui permettent de nombreuses choses, ça rajouterait un côté fun à faire des pack opening de friandise

**Comment by BastLast** [comment]
J'adore l'idée !! Je suis sur mobile mais go le mettre dans lez coups de cœur

**Comment by niqore** [comment]
Friandises pour les animaux ou les joueurs?

**Reply by BastLast** [reply]
Animaux pr moi
